### ISSUE TITLE
lineage ?

### ISSUE THREAD

**Issue by StripedyApe** [wicknix/SeaLion#12]
This is kind of trivial but ... the active tab highlight color looks to be the system highlight color, which is sensible but on the browser it's kind of garish so I went to darken it up a bit ... and ran into a maze. There's recipes for about ten different versions of firefox, pale moon, basilisk, sea monkey, and about ten more. Which one is sea lion descended from, so I can at least have a fighting chance at finding userchrome, usercontent settings that might work ?  

And you're going to hate this idea but oh well :) Remember the Netscape throbber with the shooting stars? A swimming seal throbber could be kind of cool ... /me runs for cover :-)

**Comment by wicknix** [comment]
Sorry for the delay. Been busy. It's derived from Netscape 9 / SeaMonkey 2.

Throbbers are theme dependant. Each theme can or will change the throbber if it's included in the theme. So each theme would have to be modified again to include it. It's doable though.

**Comment by StripedyApe** [comment]
hi, thank you. No problem, I'm pretty sure the world won't end because one color in the browser displeases me :)

Maybe a small help to someone tho, I'm between a rock and a hard place. Found that there was another place where the user colors were being enforced - it was one line in about:config. Changed that and then we picked up any other changes I set. However, the system colors were all better except for this one. Great :-) So I went back to system colors then tried to change that blink-tag robin's egg blue in the Mac setup. No dice. Jonny Ive owns my computer and sets it up the way he wants, to hell with me. 

That garish color is hidden somewhere in the Mac setup but it would/will take a bunch of work to find it, I bet. For now, I live with garish.

Thanks for the info on the basis, I hope that helps with modifying extensions to work. Another thing that may help someone : I was struggling my rear off trying to make really simple changes then finally found there's something wrong with my zipping technique. Using Mac default tools, if I unzip an xpi file, make NO changes, then rezip it, the browser will say it's corrupted and can't be installed. So that's possibly something people can look at if they are trying to modify extensions but having no luck with even basic changes. Make sure your zipup operation is working correctly. 

I still hope you rebrand and fixup Ambassador as SeaIrchin tho :) 

thanks again, this thing is a big help in keeping el capitan alive. 

https://imgs.xkcd.com/comics/team_chat.png

**Comment by StripedyApe** [comment]
A little update for people trying to add other extensions : yes it was my zipping method. So if you have unexplained diffic ulties with just adding a browser description then rezipping, that's one place to look. 

Added the description from the greasemonkey that was in SeaLion Extras to Overbite from floodgap, it worked perfectly and now SeaLion can browse gopher sites :-)

You might consider adding that ? Cameron is a long-time Mac Wizard and all-around good guy. And gopher is good, too!

**Comment by wicknix** [comment]
Yeah some compression tools mess with repacking the xpi. Overbite is already in the sealion extras zip btw. ;-)

**Comment by StripedyApe** [comment]
On Thu, 24 Oct 2024 12:29:46 -0700
wicknix ***@***.***> wrote:

Hi Wick,

Not sure if you are still a macker? I am not, I'm an Irix-lover but they never made an irix laptop so here I am with an elderly el cap macbook. 

*And* because that seems to be a problem, which is why I came across Sea Lion, I stuck up a local newsgroup where I keep track of stuff that works, stuff I've made to work, and so on. 

shanghairuipu.com 119 

no registration or any of that crap. If you have anything helpful or know any other el cap lunatics, it's there for sharing experiences. 

cheers, and thanks for sea lion, I use it all the time :)
